src/app/services/neeto-cal.service.ts

```
@@ -0,0 +1,52 @@
+export class NeetoCalService {
+  private isScriptLoaded = false;
+
+  constructor(@Inject(PLATFORM_ID) private platformId: object) {}
+
+  initialize(): void {
+    if (!isPlatformBrowser(this.platformId)) {
+      return;
+    }
+    (window as any).neetoCal = (window as any).neetoCal || {
+      embed: function() {
+        ((window as any).neetoCal.q = (window as any).neetoCal.q || []).push(arguments);
```

ESLint

[error] Use the rest parameters instead of 'arguments'.

(prefer-rest-params)